common/build.rs

```diff
@@ -49,10 +52,11 @@ fn main() {
         "rust swig expand time: {}",
         expand_time.as_secs() as f64 + (expand_time.subsec_nanos() as f64) / 1_000_000_000.
     );
-    println!("cargo:rerun-if-changed=src/lib.rs.in");
-    println!("cargo:rerun-if-changed=src/lib.rs");
+    println!("cargo:rerun-if-changed=../common/lib.rs.in");
```

Comment:
I have to assume that even though `build.rs` is in `common`, its working directory is still `android` (I assume because that's where the `Cargo.toml` is), because if I remove the `../common/`, it no longer works as expected

Reply:
I totally didn't expect that. Thanks for letting me know.